src/components/NinjaAPI.js

```diff
@@ -0,0 +1,115 @@
+// import '../Styles/Css/App.css';
+// import { useEffect, useState } from 'react';
+
+// const category = 'success';
+// const apiKey = 'HDkAqbGgWzD4ApoIUqCl1Q==77tdi7Q7T8rjLfPB';
+
+// function QuoteApi() {
+//   const [quote, setQuotes] = useState([]);
+//   const [hasError, setHasError] = useState(false);
+//   const [isLoading, setIsLoading] = useState(false);
+
+//   useEffect(() => {
+//     const fetchData = () => {
+//       setIsLoading(true);
+//       fetch(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
+//         method: 'GET',
+//         headers: {
+//           'X-Api-Key': apiKey,
```

Comment:
- [ ] Please remove commented code as they make your code base look untidy and difficult to maintain 👍